ibm_catalog.json

```diff
@@ -215,15 +225,16 @@
                 }
               ],
               "optional": true,
-              "on_by_default": true,
-              "version": "^v2.0.6"
+              "on_by_default": false,
+              "version": "v3.0.7"
             },
             {
-              "name": "testing-deploy-arch-ibm-observability",
-              "id": "daf2cf91-2384-4205-b7f4-621299a4896d-global",
-              "version": "^1.3.0",
+              "name": "deploy-arch-ibm-observability",
+              "description": "Cloud Automation for Observability supports configuring IBM Cloud Observability resources for logging, monitoring and activity tracker event routing.",
```

Comment:
This needs to be a reason to enable it as it applies to orchestrate something like:
>Enable to provision and configure IBM Cloud Logs, Cloud Monitoring, Metrics routing and Activity Tracker event routing for analysing logs and metrics generated by the watson Orchestrate instance

I don't know if that description is true, but something like it....